src/types/onyx/OriginalMessage.ts

```diff
@@ -15,24 +15,23 @@ type IOUDetails = {
     currency: string;
 };
 
+type IOUMessage = {
+    /** The ID of the iou transaction */
+    IOUTransactionID?: string;
+    IOUReportID?: number;
```

Comment:
Can you double check that it isn't a string?

Reply:
It's number , confirmed it on few places
<img width="988" alt="image" src="https://github.com/Expensify/App/assets/25230417/80ec0534-cfb0-4913-87cf-a8a046a15e03">
<img width="409" alt="image" src="https://github.com/Expensify/App/assets/25230417/16ed30cc-6bef-4f8d-95fa-20fc141dc686">